packages/app/components/navigation/Navigation.tsx

```diff
@@ -1,24 +1,47 @@
 import React from 'react';
-import { View, Text, TouchableOpacity, SafeAreaView } from 'react-native';
+import { View, Text, SafeAreaView, TouchableOpacity } from 'react-native';
 import { AuthStateListener } from '../../auth/AuthStateListener';
-import { Drawer } from './Drawer';
 import useCustomStyles from 'app/hooks/useCustomStyles';
 import { useIsMobileView } from 'app/hooks/common';
 import { useNavigate } from 'app/hooks/navigation';
 import { useAuthUser } from 'app/hooks/user/useAuthUser';
 import { NavigationList } from './NavigationList';
 import { Button } from 'tamagui';
 import SVGLogoComponent from '../../components/logo';
-import { Platform } from 'react-native';
-import { Image } from 'solito/image';
+import { Drawer } from 'expo-router/drawer';
```

Comment:
I believe we will need to extract this into a drawer.native.tsx and drawer.web.tsx files so this doesn't crash on next js side